src/primitives/polyline/styled.rs

```diff
@@ -457,6 +558,44 @@ mod tests {
         );
     }
 
+    #[test]
+    fn translated_bounding_box() {
+        let by = Point::new(10, 12);
+        let pl = Polyline::new(&PATTERN).translate(by);
+
+        assert_eq!(
+            pl.into_styled::<Rgb565>(PrimitiveStyle::with_stroke(Rgb565::RED, 1))
```

Comment:
Is the additional `Rgb565` required?
```suggestion
            pl.into_styled(PrimitiveStyle::with_stroke(Rgb565::RED, 1))
```
This question also applies to the other assertions.